sites/fast-website/src/app/components/content-placement-container/content-placement-container.styles.ts

```diff
@@ -47,7 +48,7 @@ export const ContentPlacementContainerStyles = css`
         }
     }
 
-    @media screen and (min-width: 1000px) {
+    @media screen and (min-width: ${siteBreakpoints[4]}) {
```

Comment:
Readability: see comments above above.

Reply:
Reverting this to fixed numbers for now.  Proposal coming forth on this.  See Chris' comment below.